{{cookiecutter.project_slug}}/.config/profiles.yml

```diff
@@ -1,14 +1,90 @@
         user: {% raw %}"{{ env_var('DBT_USER') }}"{% endraw %}
-        role: {{ cookiecutter.snowflake_role.upper() }}
+
+        # Remember to copy this file to ~/.dbt/ first
+        # Set variable DBT_PASSWORD in your CI or update this to your database password
         password: {% raw %}"{{ env_var('DBT_PASSWORD') }}"{% endraw %}
-        database: {% raw %}"{{ env_var('DBT_DATABASE') }}"{% endraw %}
-        warehouse: {{ cookiecutter.snowflake_warehouse.upper() }}
+
+        # Additional Snowflake options
+        # Find out more about Snowflake config options here:
+        # https://docs.getdbt.com/reference/warehouse-profiles/snowflake-profile
+
+        # Keypair authentication use instead of username and password
+        # private_key_path: {% raw %}"{{ env_var('DBT_KEY_PATH') }}"{% endraw %}
+        # private_key_passphrase: {% raw %}"{{ env_var('DBT_KEY_PASSPHRASE') }}"{% endraw %}
+
+        # Set the DBT_SNOWFLAKE_ACCOUNT in your CI or update this to the actual Snowflake account like abc12345]
+        account: {% raw %}"{{ env_var('DBT_SNOWFLAKE_ACCOUNT') }}"{% endraw %}
+
+        # Set the DBT_SNOWFLAKE_ROLE in your CI or update this to the actual role you use in Snowflake
+        role: {% raw %}"{{ env_var('DBT_SNOWFLAKE_ROLE') }}"{% endraw %}
+
+        # Set the DBT_SNOWFLAKE_WAREHOUSE in your CI or update this to the actual Warehouse you use in Snowflake
+        warehouse: {% raw %}"{{ env_var('DBT_SNOWFLAKE_WAREHOUSE') }}"{% endraw %}
+
+        client_session_keep_alive: False
+        # query_tag: my_tag
+        {%- endif %}
+
+        {%- if cookiecutter.adapter == "bigquery" %}
+        # Additional BigQuery options
+        # Find out more about BigQuery config options here:
+        # https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile
+
+        method: oauth
+        project: [GCP project id]
+        dataset: [the name of your dbt dataset] # You can also use "schema" here
+        timeout_seconds: 300
+        location: US # Optional, one of US or EU
+        priority: interactive
+        retries: 1
+        {%- endif %}
+
+        {%- if cookiecutter.adapter == "redshift" %}
+        # Additional Redshift options
+        # Find out more about Redshift config options here:
+        # https://docs.getdbt.com/reference/warehouse-profiles/redshift-profile
+
+        # Set the TEST_DATABASE in your CI or update this to the actual database name
+        dbname: {% raw %}"{{ env_var('TEST_DATABASE') }}"{% endraw %}
         schema: {{ cookiecutter.target_schema }}
-        threads: 8
+
+        # Set variable DBT_USER in your CI or update this to your database username name
+        user: {% raw %}"{{ env_var('DBT_USER') }}"{% endraw %}
+
+        # Remember to copy this file to ~/.dbt/ first
+        # Set variable DBT_PASSWORD in your CI or update this to your database password
+        password: {% raw %}"{{ env_var('DBT_PASSWORD') }}"{% endraw %}
```

Comment:
Suggesting raw passwords in this file is not a good idea, we need this file committed for the CI pipeline.

Reply:
oh, I was copying from other lines without understanding
`{% raw %}"{{ env_var('DBT_PASSWORD') }}"{% endraw %}`
will actually put the psw in the file?  We are not asking for it.
How do I just reference the CI var is it just 
`env_var('DBT_PASSWORD')`

Reply:
No, it won't put the password in the file, it will just read it from an env var.